docs/standard/simd.md

```diff
@@ -0,0 +1,104 @@
+---
+title: SIMD in .NET
+description: This tutorial demonstrates how to use SIMD in C#.
+author: FIVIL
+#ms.author: [MICROSOFT ALIAS OF INTERNAL OWNER]
+ms.date: 07/05/2019
+#ms.topic: [TOPIC TYPE]
+#ms.prod: [PRODUCT VALUE]
+ms.technology: dotnet-standard
+---
+
+# Overview
+
+SIMD (Single instruction, multiple data) provides the functionality to do multiple processes on a single core to achieve peak performance. In .NET there is set of SIMD-enabled types under <xref:System.Numerics> namespace. SIMD operations can be parallelized at the hardware level. That increases the throughput of the vectorized computations, which are common in mathematical, scientific, and graphics apps.
+
+## .NET SIMD-enabled types
+
+The .NET SIMD-enabled types include the following:
```

Comment:
I'm not sure saying they are `SIMD-enabled` is correct. Its that certain exposed APIs on the types may be SIMD accelerated.